katalog/istio/README.md

```diff
@@ -1,5 +1,22 @@
 # Istio
 
+- [Istio](#istio)
+  - [Requirements](#requirements)
+  - [Modular](#modular)
+    - [Minimal installation](#minimal-installation)
+    - [Telemetry](#telemetry)
+      - [Service Monitors](#service-monitors)
+        - [Note](#note)
+    - [Egress Gateway](#egress-gateway)
+    - [Citadel](#citadel)
+        - [Note](#note-1)
+    - [Sidecar injector](#sidecar-injector)
+        - [Note](#note-2)
+    - [Kiali](#kiali)
+      - [Configuration](#configuration)
+
+___
+
 Istio addresses the challenges developers and operators face as monolithic applications transition towards a
 distributed microservice architecture. To see how, it helps to take a more detailed look at Istio’s service mesh.
```

Comment:
The last sentence is weird, perhaps something like this would be better?:
"Istio addresses the challenges developers and operators face as monolithic applications transition towards a
distributed microservice architecture, in the following sections we got deeper into Istio service mesh's details to see how this is achieved."